tests/integration/security/authorization_test.go

```diff
@@ -906,54 +831,114 @@ func TestAuthorization_TCP(t *testing.T) {
 					ExpectAllowed: expectAllowed,
 				}
 			}
+			ns := apps.Namespace1
+			ns2 := apps.Namespace2
+			src0 := apps.A.Match(echo.Namespace(ns.Name()))
+			src1 := apps.B.Match(echo.Namespace(ns.Name()))
+			src2 := apps.E.Match(echo.Namespace(ns.Name()))
+			src3 := apps.C.Match(echo.Namespace(ns2.Name()))
+
+			dst0 := apps.B.Match(echo.Namespace(ns.Name()))
+			dst1 := apps.C.Match(echo.Namespace(ns.Name()))
+			dst2 := apps.D.Match(echo.Namespace(ns.Name()))
+			dst3 := apps.E.Match(echo.Namespace(ns.Name()))
+			dst4 := apps.A.Match(echo.Namespace(ns.Name()))
+			t.NewSubTest(fmt.Sprintf("to %s %s %s %s %s", dst0[0].Config().Service,
+				dst1[0].Config().Service, dst2[0].Config().Service, dst3[0].Config().Service, dst4[0].Config().Service)).
+				Run(func(t framework.TestContext) {
+					policy := tmpl.EvaluateAllOrFail(t, map[string]string{
+						"Namespace":  ns.Name(),
+						"Namespace2": ns2.Name(),
+						"dst0":       dst0[0].Config().Service,
+						"dst1":       dst1[0].Config().Service,
+						"dst2":       dst2[0].Config().Service,
+						"dst4":       dst4[0].Config().Service,
+						"src0":       src0[0].Config().Service,
+						"src1":       src1[0].Config().Service,
+					}, file.AsStringOrFail(t, "testdata/authz/v1beta1-tcp.yaml.tmpl"))
+					t.Config().ApplyYAMLOrFail(t, "", policy...)
+					cases := []rbacUtil.TestCase{
+						// The policy on workload dst0 denies request with path "/data" to port 8091:
+						// - request to port http-8091 should be denied because both path and port are matched.
+						// - request to port http-8092 should be allowed because the port is not matched.
+						// - request to port tcp-8093 should be allowed because the port is not matched.
+						newTestCase(src0, dst0, "http-8091", false, scheme.HTTP),
```

Comment:
why are we changing the port from http-8090 to http-8091, http-8091 to http-8092, etc.? This makes the diff really hard to review, can we just stick to the old ports, the new vm test should not affect the current tests.